### ISSUE TITLE
Need to test hyper parameters to determine optimal tuning

### ISSUE THREAD

**Issue by LTiede4** [TheCommunityGit/PINN_Navier_Stokes#1]
We can test the following hyper parameters:
   - batch_size       [512, 1024, 2048]
   - epochs             [32, 64, 128, 256, 512]
   - learning rate    [1e-4, 5e-4, 1e-3, 5e-3]
 
For the physics influence in the loss function
   - nu
   - rho

(not as high of a priority):
   - num_layers      [6, 8, 10, 12]
   - hidden_size     [32, 64, 128]
